doc/api/cli.md

```diff
@@ -443,6 +443,55 @@ Affects the default output directory of:
 * [`--heap-prof-dir`][]
 * [`--redirect-warnings`][]
 
+### `--disable-warning=code-or-type`
```

Comment:
Can we make this doc-only experimental? In case we want to remove it, we shouldn't have to wait for a major.

Reply:
What would I need to do in order to make it "doc-only experimental"? 

I think its a good idea to do so 👍

Reply:
Something like `Stability: 1.0 - Early development`. Example: https://github.com/nodejs/node/blob/main/doc/api/cli.md#--env-fileconfig

Reply:
done